src/test/kotlin/no/skatteetaten/aurora/boober/service/AuroraConfigHelper.kt

```diff
@@ -85,3 +90,17 @@ private fun convertFileToJsonNode(file: File): JsonNode? {
     return mapper.readValue(file, JsonNode::class.java)
 }
 
+fun modifyCommandIfRoute(json: JsonNode): OpenshiftCommand {
```

Comment:
This really has nothing to do with AuroraConfig. Consider moving it to another/new utility class. Separation of concerns.